src/components/FontsLoader.jsx

```diff
@@ -30,11 +30,11 @@ export default class FontsLoader {
             .filter(k => k.indexOf(FontsLoader.PREFIX) > -1)
             .forEach(k => {
                 const name = k.split(FontsLoader.PREFIX)[1];
-                this.addFont(name, localStorage.getItem(k));
+                this.addFont(name, localStorage.getItem(k), true);
```

Comment:
`for each`  are not optimal in this new behavior.
but to avoid an initialization on each fonts, we could just force the last one.
```js
        .forEach((k, i, arr)=>{
            this.addFont(name, localStorage.getItem(k), i===arr.length-1);
```
maybe :)